### PR TITLE
chore(packages): update for consistency

### PR DESCRIPTION
**Pull request by awentzel**
<body>
Updates include code coverage, making package.json files consistent, and updating to use URL for contact rather than a deprecated EMAIL address.
<footer>

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

**Comment by awentzel** [comment]
Note that while the code coverage minimum thresholds must be 50, I've raised the bar. For example, if code coverage for statements, lines, branches, or functions are already at 75, why not just reset the baseline to have a higher bar.

File: packages/fast-animation/package.json [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
-  "main": "dist/index.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
-  "author": {
-    "name": "Microsoft",
-    "email": "fwq@microsoft.com"
-  },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-animation/package.json]: I don't believe this is a field that is part of the package.json [spec](https://docs.npmjs.com/files/package.json). I also think README.md at the root of a package is the default.

File: packages/fast-components-class-name-contracts-base/package.json [hunk 2]
```diff
@@ -5,12 +5,14 @@
...
     "name": "Microsoft",
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-components-class-name-contracts-base/package.json]: Again, this is the default and readme key in package.json is not part of the [spec](https://docs.npmjs.com/files/package.json).

File: packages/fast-components-class-name-contracts-msft/package.json [hunk 3]
```diff
@@ -5,12 +5,14 @@
...
     "name": "Microsoft",
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-components-class-name-contracts-msft/package.json]: Not part of the [spec](https://docs.npmjs.com/files/package.json).

File: packages/fast-components-react-base/package.json [hunk 4]
```diff
@@ -5,12 +5,14 @@
...
     "name": "Microsoft",
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-components-react-base/package.json]: Not part of the [spec](https://docs.npmjs.com/files/package.json)

File: packages/fast-components-react-msft/package.json [hunk 5]
```diff
@@ -33,6 +33,12 @@
   },
   "jest": {
     "collectCoverage": true,
+    "global": {
```
Comment (janechu) [inline packages/fast-components-react-msft/package.json]: Should this be in coverageThreshold?

File: packages/fast-form-generator-react/package.json [hunk 6]
```diff
@@ -2,15 +2,17 @@
...
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
-  "version": "1.8.0",
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-form-generator-react/package.json]: Not part of the [spec](https://docs.npmjs.com/files/package.json).

File: packages/fast-form-generator-react/package.json [hunk 7]
```diff
@@ -25,10 +27,14 @@
     "unit-tests:update": "jest --updateSnapshot",
     "unit-tests:watch": "jest --watch"
   },
-  "license": "MIT",
-  "readme": "README.md",
   "jest": {
     "collectCoverage": true,
+    "global": {
```
Comment (janechu) [inline packages/fast-form-generator-react/package.json]: Should this belong in coverageThreshold?

File: packages/fast-sketch-library/package.json [hunk 8]
```diff
@@ -6,12 +6,14 @@
...
     "name": "Microsoft",
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
+  "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
+  "readme": "README.md",
```
Comment (janechu) [inline packages/fast-sketch-library/package.json]: Not part of the [spec](https://docs.npmjs.com/files/package.json).

File: packages/fast-jss-utilities/package.json [hunk 9]
```diff
@@ -5,14 +5,14 @@
...
-    "email": "fwq@microsoft.com"
+    "url": "https://discord.gg/FcSNfg4"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Microsoft/fast-dna.git"
   },
-  "readme": "README.md",
+
```
Comment (janechu) [inline packages/fast-jss-utilities/package.json]: nit: extra line

**Pull request merged**